server/src/test/java/io/spine/server/entity/VisibilityGuardShould.java

```diff
@@ -174,4 +182,27 @@ private HiddenRepository() {
             super();
         }
     }
+
+    private static class EntityWithInvalidColumns extends AbstractVersionableEntity<String, Any> {
```

Comment:
Please migrate to `...TestEnv`-based approach. See `server` module source code for numerous examples.